waku/waku_rln_relay/group_manager/on_chain/group_manager.nim

```diff
@@ -39,6 +39,7 @@ contract(RlnStorage):
   proc MEMBERSHIP_DEPOSIT(): Uint256 {.pure.}
   proc members(idCommitment: Uint256): Uint256 {.view.}
   proc idCommitmentIndex(): Uint256 {.view.}
+  proc deployedBlockNumber(): Uint256 {.view.}
```

Comment:
Looks nice indeed! However, I wonder if we could add comments above each of these procs to elaborate more about their purposes.

Reply:
will address in a follow up pr :)